### PR TITLE
OPENEUROPA-1328: Media entity browser handling for AV Portal media.

### PR DESCRIPTION
**Pull request by sergepavle**
The scope of this ticket is to expose the AV Portal media entities to the generic entity browser in the demo module.

Todo:

- [x] Create entity browser widget for "creating" AV Portal media entities
- [x] Adjust the test that selects media via the entity browser to select also existing AV Portal entities
- [x] Test the entity browser widget that creates and selects the AV Portal entity

### CODE REVIEW

File: tests/features/media-avportal-entity-browser.feature [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@api
+Feature: Media AV portal with entity browser.
+  In order to be able to showcase the media and entity browser features for managing AV Portal video
+  As a site editor
+  I want to see the entity browser widget for adding and reusing AV Portal video media entities.
+
+  @javascript @av_portal
+  Scenario: The node adding form should contain entity browser widget with possibility to add new and reuse existing AV Portal video.
+    Given I am logged in as a user with the "create oe_media_demo content,create av_portal_video media,access media_entity_browser entity browser pages" permission
```
[the remark this answers is not in the document]
Comment (upchuk) [reply tests/features/media-avportal-entity-browser.feature]: @voidtek Why? There is only one scenario

File: tests/features/media-avportal-entity-browser.feature [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    And I visit "node/add/oe_media_demo"
+    And I fill in "Title" with "Media demo"
+    And I click the fieldset "Media browser field"
+    When I press the "Select entities" button
```
[the remark this answers is not in the document]
Comment (upchuk) [reply tests/features/media-avportal-entity-browser.feature]: Not really. The pressing is the trigger that should then then call for the "then". So it's correct.

File: tests/features/media-avportal-entity-browser.feature [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    Then I should see entity browser modal window
+    When I click "View"
+    And I select the "Midday press briefing from 25/10/2018" media entity in the entity browser modal window
+    And I press the "Select entities" button
+    And I press the "Save" button
+    Then I should see the AV Portal video "Midday press briefing from 25/10/2018"
+    # Cleanup of the media entity.
+    And I remove the media "Midday press briefing from 25/10/2018"
```
[the remark this answers is not in the document]
Comment (upchuk) [reply tests/features/media-avportal-entity-browser.feature]: I agree. Will create the follow up for this.

File: tests/features/media-avportal-entity-browser.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (voidtek) [reply tests/features/media-avportal-entity-browser.feature]: So if it's the case, please insert a `then` before.
According the documentation http://docs.behat.org/en/v2.5/quick_intro.html

> Each part of the scenario - the context, the event, and the outcome - can be extended by adding the And or But keyword:
> 
> Scenario: Some description of the scenario
>   Given [some context]
>     And [more context]
>    When [some event]
>     And [second event occurs]
>    Then [outcome]
>     And [another outcome]
>     But [another outcome]

[the remark this answers is not in the document]
Comment (drupol) [reply tests/features/media-avportal-entity-browser.feature]: Too late, it's merged already.

**Pull request merged**